Bio/Align/Applications/_Guidance2.py

```diff
@@ -0,0 +1,157 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 by Rob Gilmore and Shaurita Hutchins. All rights reserved.
+# Based on ClustalOmega wrapper copyright 2011 by Andreas Wilm.
+#
+# Wrapper for Guidance2 by Rob Gilmore (2017).  http://guidance.tau.ac.il/ver2/
+# Used _ClustalOmega.py as template.
+#
+# This code is part of the Biopython distribution and governed by its
+# license.  Please see the LICENSE file that should have been included
+# as part of this package.
+"""Command line wrapper for the multiple alignment program, GUIDANCE2.
```

Comment:
Please insert a blank line after the one line module summary as per PEP257 https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings